paper/paper.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Vector: creating and manipulating jagged arrays of Lorentz vectors"
+tags:
+  - Python
+  - vector algebra
+  - high energy physics
+authors:
+  - name: Henry Schreiner
+    orcid: 0000-0002-7833-783X
+    equal-contrib: true
+    affiliation: 1
+  - name: Jim Pivarski
+    orcid: 0000-0002-6649-343X
+    equal-contrib: true
+    corresponding: true
+    affiliation: 1
+  - name: Saransh Chopra
+    orcid: 0000-0003-3046-7675
+    equal-contrib: true
+    affiliation: 1
+
+affiliations:
+  - name: Princeton University
+    index: 1
+date: 16 August 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+Vector algebra is a crucial component of data analysis pipelines in high energy
+physics, enabling physicists to transform raw data into meaningful results that
+can be visualized. Given that high energy physics data is not uniform, the
+vector algebra frameworks or libraries are expected to work readily on
+non-uniform or jagged data, allowing users to perform operations on an entire
+jagged array in minimum passes. Furthermore, optimizing memory usage and
+processing time has become essential with the increasing computational demands
+at the LHC. Vector is a Python library for creating and manipulating 2D, 3D,
+and Lorentz vectors, especially arrays of vectors, to solve common physics
+problems in a NumPy-like [@harris:2020] way. The library enables physicists to
+operate on high energy physics data in a high level language without
+compromising speed. The library is already in use at LHC and is a part of
+frameworks, like Coffea [@Gray:2023], employed by physicists across multiple
+high energy physics experiments.
+
+# Statement of need
+
+Vector is currently the only Lorentz vector library providing a Pythonic
+interface but a C++ (through Awkward Array [@Pivarski:2018]) computational
+backend. Vector integrates seamlessly with the existing high energy physics
+ecosystem and the broader scientific Python ecosystem, including libraries like
+Dask [@rocklin:2015] and Numba [@lam:2015]. The library implements a variety of
+backends for several purposes. Although vector was written with high energy
+physics in mind, it is a general-purpose library that can be used for any
+scientific or engineering application. The library houses 3+2 numerical
+backends for experimental physicists and 1 symbolic backend for theoretical
```

Comment:
I'm not sure what `3+2` numerical backends means. It could be worthwhile to present all of the numerical backends and their purposes in a bulleted list. A strong point is the diversity of _types_ of backends, from scalars (builtin), to collection types (NumPy and Awkward), to symbolic (SymPy).